Client/Ecosia/L10N/en.lproj/Ecosia.strings

```diff
@@ -157,6 +157,9 @@
 "Trees planted by the Ecosia community" = "Trees planted by the Ecosia community";
 "dedicated to climate action" = "dedicated to climate action";
 "Active projects" = "Active projects";
+"%@+" = "%@+";
+"60+" = "60+";
+"30+" = "30+";
```

Comment:
I don't think it makes sense to use the actual numbers in the translations, could we maybe just have the generic `"%@+" = "%@+";` which defines where and what to use as the plus sign and set the numbers outside?

Reply:
Yeah of course we can 👍 . Went after the simplest solution but will certainly update 😉 . There's also a chance to save a couple of translations strings as  `"%@+" = "%@+";` could be used elsewhere like the `250_000_000+`